exalearn-api/ExaLearn.Dal/Interfaces/IUserRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ExaLearn.Dal.Entities;
+
+namespace ExaLearn.Dal.Interfaces
+{
+    public interface IUserRepository : IGenericRepository<User>
```

Comment:
If you really need the files, please add some comments where and when could be possible to use